perception/autoware_elevation_map_loader/src/elevation_map_loader_node.cpp

```diff
@@ -294,9 +302,18 @@ void ElevationMapLoaderNode::receiveMap()
     concatenatePointCloudMaps(pointcloud_map, result.get()->new_pointcloud_with_ids);
   }
   RCLCPP_DEBUG(this->get_logger(), "Pointcloud map receiving process has been finished");
+
+  // check for empty point cloud
+  // TODO(youtalk): add unit test for empty point cloud handling
+  if (pointcloud_map.data.empty() || pointcloud_map.width == 0 || pointcloud_map.height == 0) {
+    RCLCPP_WARN(this->get_logger(), "Empty pointcloud_map received after concatenation");
```

Comment:
@youtalk 
Thank you for your PR!

[IMO]
Consider using `RCLCPP_WARN_THROTTLE` to avoid cluttering the logs.